quickemu

```diff
@@ -1629,9 +1687,13 @@ else
             USB_CONTROLLER="${2}"
             shift;
             shift;;
-          -extra_args|--extra_args)
```

Comment:
As  'args'  has been coded as an array instead of a string, this means that all the items given as arguments are to be space separated.  As shellcheck keeps pointing out ...  

What happens in the old code is that  when supplying  `--extra_args "-device  tpm-tis,tpmdev=tpm0"`  for example, only the first bit will be read.  The shift, which doesn't need the repeated semi-colons, as highlighted by you above on line 1630,  will simply move second part of the extra args into the next positional.

Adding the shortened version of  --extra_args as  --xa   saves alot of typing.